public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_monitor/azure_monitor_datasource.ts

```diff
@@ -105,7 +110,7 @@ export default class AzureMonitorDatasource {
       },
     });
 
-    const result = [];
+    const result: any[] = [];
```

Comment:
Shouldn't you be able to use `DataQueryResponse` instead of `any`? See [DatasourceApi](https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/types/datasource.ts#L145). You maybe want to extend that and use the new plugin export api, like for example Elasticsearch:
[datasource](https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/elasticsearch/datasource.ts#L14)
[module](https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/elasticsearch/module.ts#L11)